src/pages/index.tsx

```diff
@@ -233,9 +233,39 @@ export default function Home() {
         </div>
       </div>
 
+      <div className="py-4 bg-dark">
+        <div className="container-md">
+          <Row>
+            <Column>
+              <h2 className="text-white">Open-source. Forever.</h2>
+              <p className="text-white ">
+                PrairieLearn began life as open-source software, and we&apos;re
+                committed to making sure it stays that way. With an active
+                developer community of professors, course staff, and students,
+                PrairieLearn gets better all the time.
+              </p>
```

Comment:
I moved this back to the homepage. I think this is a pretty core part of PL, and I don't want to bury this in the "about" page.